src/terraboot/infra.clj

```diff
@@ -29,7 +30,7 @@
                                          :min-number-of-public-slaves 1
                                          :max-number-of-public-slaves 1
                                          :public-slave-instance-type "t2.medium"
-                                         :public-slave-elb-listeners [{:lb_port 443 :lb_protocol "https" :port 80 :protocol "http"}
+                                         :public-slave-elb-listeners [{:lb-port 443 :lb-protocol "https" :port 80 :protocol "http" :cert-name "c512707d-bbec-4859-ab22-0f5fbad62a22"}
```

Comment:
this looks like the place for all of our specific config, so I'm guessing this UUID is fine?